integration-tests/browser_integration_test.go

```diff
@@ -65,8 +67,18 @@ func TestBrowserIntegration(t *testing.T) {
 
 	// standaloneOrder is an order that will be sent to the network by the
 	// standalone node.
-	ethClient := ethclient.NewClient(ethRPCClient)
-	standaloneOrder := scenario.CreateZRXForWETHSignedTestOrder(t, ethClient, makerAddress, takerAddress, wethAmount, zrxAmount)
+	standaloneOrder := scenario.NewSignedTestOrder(t, orderopts.SetupMakerState(true))
```

Comment:
I don't think we can put it in the `init` functions since the requirements for each test order may be different depending on the test. I also don't want to make `SetupMakerState` default to true for reasons mentioned in my other comment.